### PR TITLE
Fixed Documentation warnings from PEcAn.emulator

### PR DESCRIPTION
**Pull request by nanu1605**

### CODE REVIEW

**Comment by moki1202** [comment]
@nanu1605 just mentioning here the arguments that still need to be documented-

- [ ] myY
- [ ] maxval
- [ ] ic
- [ ] rate
- [ ] xnew
- [ ] pos.check
- [ ] SSnew
- [ ] ycurr
- [ ] ynew

**Comment by nanu1605** [comment]
I was searching for their description but wasn't able to find them. I was about to ask in the #cran channel regarding their description.

File: modules/emulator/R/ldinvgamma.R [hunk 1]
```diff
@@ -4,9 +4,9 @@
 ##' @title ldinvgamma
 ##' @export
 ##'
-##' @param x
-##' @param shape
-##' @param scale
+##' @param x Name of variable to plot on X axis
+##' @param shape shape of points
+##' @param scale scale, NULL by default
```
Comment (infotroph) [inline modules/emulator/R/ldinvgamma.R:9]: A lot of the functions in this package that take `x` arguments aren't plotting anything, but rather computing densities of statistical distributions, so "name of variable to plot" isn't the right description. What to replace it with will differ from one function to another; here's a suggestion for this one, but please check the others too.

```suggestion
##' @param x vector of quantiles
##' @param shape,scale shape and scale parameters for the inverse Gamma distribution
```

**Comment by nanu1605** [comment]
@infotroph @moki1202 Is something missing in `data.land`? the CI checks are failing because of it

**Comment by moki1202** [comment]
@nanu1605 I don't think `data.land` would cause any `CI check` failure atleast in this PR. Maybe I missed something. Can you mention the lines you're referring to?

File: modules/emulator/R/GaussProcess.R [hunk 2]
```diff
@@ -2,20 +2,21 @@
 ##' @title GaussProcess
 ##' @export
 ##'
-##' @param x
-##' @param y
-##' @param isotropic
+##' @param x Name of variable to plot on X axis
+##' @param y Name of variable to plot on Y axis
```
Comment (mdietze) [inline modules/emulator/R/GaussProcess.R:6]: I don't think this function plots anything. Y is our dependent variable (I assume a vector) and X is our set of independent variables (I assume a matrix?)

File: modules/emulator/R/calcSpatialCov.R [hunk 3]
```diff
@@ -2,5 +2,8 @@
 ##' @title calcSpatialCov 
 ##' @export
 ##'
+##' @param x Name of variable to plot on X axis
+##' @param ... Adiitional arguments
```
Comment (mdietze) [inline modules/emulator/R/calcSpatialCov.R:6]: additional is spelled wrong

File: modules/emulator/R/calcSpatialCov.R [hunk 4]
```diff
@@ -2,5 +2,8 @@
 ##' @title calcSpatialCov 
 ##' @export
 ##'
+##' @param x Name of variable to plot on X axis
```
Comment (mdietze) [inline modules/emulator/R/calcSpatialCov.R:5]: Likewise, X here isn't a plot. If you look at the two options that get called (calcSpatialCov.list and calcSpatialCov.matrix) the first argument is either a spatial distance matrix or a list of component spatial distance matrices

File: modules/emulator/R/distance.R [hunk 5]
```diff
@@ -2,8 +2,8 @@
 ##' @title distance
 ##' @export
 ##'
-##' @param x
-##' @param power
+##' @param x Name of variable to plot on X axis
```
Comment (mdietze) [inline modules/emulator/R/distance.R:5]: X here is a matrix of locations in physical or parameter space

File: modules/emulator/R/distance.matrix.R [hunk 6]
```diff
@@ -2,9 +2,9 @@
 ##' @title distance.matrix
 ##' @export
 ##'
-##' @param x
-##' @param power
-##' @param dim
+##' @param x Name of variable to plot on X axis
```
Comment (mdietze) [inline modules/emulator/R/distance.matrix.R:5]: X here is also a matrix of locations in parameter space

File: modules/emulator/R/distance12.matrix.R [hunk 7]
```diff
@@ -2,9 +2,9 @@
 ##' @title distance12.matrix
 ##' @export
 ##'
-##' @param x
-##' @param n1
-##' @param power
+##' @param x Name of variable to plot on X axis
```
Comment (mdietze) [inline modules/emulator/R/distance12.matrix.R:5]: matrix of locations in parameter space

File: modules/emulator/R/groupid.R [hunk 8]
```diff
@@ -2,6 +2,7 @@
 ##' @title groupid
 ##' @export
 ##' 
+##' @param x Name of variable to plot on X axis
```
Comment (mdietze) [inline modules/emulator/R/groupid.R:5]: matrix of parameter values

File: modules/emulator/R/minimize.GP.R [hunk 9]
```diff
@@ -97,6 +97,9 @@ gpeval <- function(xnew, k, mu, tau, psi, x, rng, splinefcns) {
 ##' @name ddist
 ##' @title ddist
 ##' @export
+##' 
+##' @param x Name of variable to plot on X axis
```
Comment (mdietze) [inline modules/emulator/R/minimize.GP.R:101]: vector of values (e.g. observations) to be evaluated by the specified probability density function

File: modules/emulator/R/minimize.GP.R [hunk 10]
```diff
@@ -97,6 +97,9 @@ gpeval <- function(xnew, k, mu, tau, psi, x, rng, splinefcns) {
...
+##' @param prior 'unif', 'IG'
```
Comment (mdietze) [inline modules/emulator/R/minimize.GP.R:102]: data.frame specifying a prior probability distribution in terms of the distribution name (distn) and first and second parameters (parama, paramb)

File: modules/emulator/R/p.R [hunk 11]
```diff
@@ -2,4 +2,7 @@
 ##' @title p
 ##' @export
 ##' 
+##' @param x Name of variable to plot on X axis
```
Comment (mdietze) [inline modules/emulator/R/p.R:5]: jump distribution

**Review by mdietze — CHANGES_REQUESTED** [review]

**Review by mdietze — APPROVED** [review]

**Pull request merged**